<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LHVDPIO" failures="1" tests="2" name="Test121" time="31.245" errors="0" timestamp="2023-02-23T11:39:46 PST">
  <testcase classname="TestNG.FacebookTestNG" name="test" time="3.291"/>
  <testcase classname="TestNG.FacebookTestNG" name="test1" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNG.FacebookTestNG.test1(FacebookTestNG.java:83)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- Test121 -->
